### ISSUE TITLE
"muteLog" option is not working

### ISSUE THREAD

**Issue by bugmaker-code** [keithbrink/amazon-mws-laravel#16]
Hi,

Thank you for providing us such great library. 
I'd flag an issue. even if I set "muteLog" is true, it's still logging in laravel 5.5

**Comment by keithbrink** [comment]
Thanks I'll take a look. When you have a sec, could you show me how you're passing the muteLog option?

**Comment by bugmaker-code** [comment]
```
$config = [
      "merchantId" => $client->sellerid,
      "marketplaceId" => $client->marketplaceId,
      "keyId" => MWS_ACCESS_KEY,
      "secretKey" => MWS_SECRET_KEY,
      "amazonServiceUrl" => $serviceURL,
      "mwsAuthToken" => $client->mwstoken,
      "muteLog" => true
 ];
```
I'm using dynamic config and `setConfig` method

**Comment by keithbrink** [comment]
Thanks, that's helpful.

**Comment by bugmaker-code** [comment]
Just FYI, when I disable `call_user_func(['Log', $loglevel], $msg)` in `AmazonCore.php` file(line #529), it looks like muteLog functionality is working properly.

**Comment by keithbrink** [comment]
Fixed in 7.0.1, thanks for reporting! The issue was the call_user_func coming before the muteLog check.

Added two tests to ensure this doesn't come up again.
